packages/x-components/src/x-modules/search-box/components/search-button.vue

```diff
@@ -176,10 +186,13 @@ you enter a search term and click the Search button, the “Looking for results
 
 _Type any term in the input field and then click the Search button to try it out!_
 
-```vue
+```vue live
 <template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work